### PR TITLE
Revert - create new bookings - writing team members to DB

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

Fixes # (issue)

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 1]
```diff
@@ -1454,7 +1455,30 @@ async function handler(
...
+              //perhaps make email fetch request to see if his locale is stored, else
+              const retObj = {
+                name: attendee.name,
+                email: attendee.email,
+                timeZone: attendee.timeZone,
+                locale: attendee.language.locale,
+              };
+              return retObj;
+            }),
+            // Have this for now until we change the relationship between bookings & team members
```
Comment (emrysal) [inline packages/features/bookings/lib/handleNewBooking.ts:1470]: Is this bit needed? also could you add it to the attendeesData? https://github.com/calcom/cal.com/pull/7756/files#diff-a7c21727d9a0d4d1aaff9d24fca9293bc28468861bc8022dfa03116908b6fae7R1432

**Review by emrysal — CHANGES_REQUESTED** [review]

File: packages/features/bookings/lib/handleNewBooking.ts (hunk 1 above)
Comment (zomars) [reply packages/features/bookings/lib/handleNewBooking.ts:1470]: Can we create a helper for this?

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 2]
```diff
@@ -1454,7 +1455,30 @@ async function handler(
...
+            ...(evt.team?.members
+              ? evt.team.members.map((member) => {
+                  return {
+                    email: member.email,
+                    name: member.name,
+                    timeZone: member.timeZone,
+                    locale: member.language.locale,
+                  };
+                })
+              : []),
```
Comment (zomars) [inline packages/features/bookings/lib/handleNewBooking.ts:1480]: Do we want ALL team members as attendees? Does this takes into account round-robin?

**Comment by zomars** [comment]
@joeauyeung What is this fixing exactly?

**Review by emrysal — APPROVED** [review]
Push team members into attendeesData array if present.

File: packages/features/bookings/lib/handleNewBooking.ts (hunk 2 above)
Comment (emrysal) [reply packages/features/bookings/lib/handleNewBooking.ts]: The flow isn't nice, but it goes from users -> availableUsers -> teamMemberPromises -> teamMembers -> team.members = teamMembers.

**Pull request merged**